src/main/java/com/datastax/loader/CqlSchemaFuzzyMatcher.java

```diff
@@ -0,0 +1,63 @@
+package com.datastax.loader;
+
+import me.xdrop.fuzzywuzzy.FuzzySearch;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Created by sebastianestevez on 12/6/16.
+ */
+
+public class CqlSchemaFuzzyMatcher {
+    public CqlSchemaFuzzyMatcher() {
+    }
+
+    public String match(String keyspace, String table, BufferedReader reader, String delimiter, CqlDelimParser cdp) {
+        String schemaString = keyspace.replaceAll("\"","")+ "." + table.replaceAll("\"","") + "(";
+        List<String> columnNames = cdp.getColumnNames();
+        try {
+            String header = reader.readLine();
+            //this didn' work
+            //System.out.println(header);
+            //List<Object> headerList = cdp.parse(header);
+            if (delimiter == null){
+                delimiter = ",";
+            }
+            List<String> headerList = Arrays.asList(header.split(delimiter));
```

Comment:
When performing a basic split, it won't account for the various CSV edge cases. Would be more robust to use a CSV parser to do the split (like Univocity, used elsewhere in this project)